hack/cve/trivy.yaml

```diff
@@ -0,0 +1,20 @@
+# Ref: https://aquasecurity.github.io/trivy/v0.33/docs/references/customization/config-file
+# Global Options
+debug: true
+# Report Options
+exit-code: 1
+format: 'sarif'
+output: 'trivy-results.sarif'
+severity:
+  - CRITICAL
+  - HIGH
+# Scan Options
+scan:
+  skip-dirs:
+    - docs
+    - hack
```

Comment:
`hack` also has code, so maybe it's useful there as well?

Reply:
Makes sense. Will remove this.

Reply:
Updated in latest changes.